nraufu/portofolio
# #landing-page: users landing page

> #### What does this PR do?
- It creates a landing page for users of the website
> #### Description of Task to be completed
- add folder structure (css,js,index.html)
- add a navbar with links to visit other pages
- add h1 tags containing small information about the developer
> #### How should this be manually tested?
- git clone the project
- open the index.html file in the browser you should see the page
> #### What are the relevant Trello stories?
- https://trello.com/c/EcWmS32m/1-user-landing-page
#### Screenshots (if appropriate)
![landing__page](https://user-images.githubusercontent.com/49249449/87689736-dcac9900-c788-11ea-8107-e8b88a8b9b80.PNG)
